### PR TITLE
Change/rsyncd template

### PR DESCRIPTION
**Pull request by stejoo**
I placed the template files in subdirectories to match their location on the target host.
I also extended the rsync confirguration to not try to compress files that are already compressed. Might save somebody some time.

### CODE REVIEW

**Comment by sbesson** [comment]
Thanks for your contribution @stejoo. The Travis CI failure is unrelated to the changes proposed in this PR. But we will want to fix it nevertheless in order to properly review the changes.

In a nutshell, the role testing infrastructure needs to be upgraded to run with Python 3/Molecule 2 - see https://github.com/ome/ansible-roles/issues/9 for the issue tracking this effort across our Ansible roles as well as a minimal set of upgrade instructions. If you had some capacity to tackle this upgrade, let us know and/or feel free to open a separate PR. Otherwise, we will try and come back to this upgrade in the upcoming days.

**Comment by stejoo** [comment]
Thanks @sbesson. I noticed the link to your issue contains an upgrade path. I'm currently hard at work for my client (which had need for an rsync server, thanks for the role :+1: ), but if I find some spare time I'll have look at the CI failure. I'm not very experienced with it unfortunately but it might be a good opportunity to learn more about it.

**Comment by stejoo** [comment]
Seems somebody else beat me to it. @manics did the good job of updating the Molecule bits.

**Comment by joshmoore** [comment]
Fixed indentation:
```
--> Executing Yamllint on files found in /home/travis/build/ome/ansible-role-rsync-server/...
    /home/travis/build/ome/ansible-role-rsync-server/tasks/main.yml
      21:3      warning  comment not indented like content  (comments-indentation)
    
An error occurred during the test sequence action: 'lint'. Cleaning up.
```

File: tasks/main.yml [hunk 1]
```diff
@@ -17,15 +17,15 @@
   become: true
   template:
     dest: /etc/rsyncd.conf
-    src: etc-rsyncd-conf.j2
+    src: etc/rsyncd.conf.j2
```
Comment (manics) [inline tasks/main.yml]: The rest of our roles use a flat hierarchy for files and templates, so it's preferable to keep this consistent. Renaming `-` to `.` in `-conf.j2` is fine though.

Comment (stejoo) [reply tasks/main.yml:20]: Fair enough, consistency is nice. Want me to change this and add it to this pull?

Comment (manics) [reply tasks/main.yml:20]: Please do.

**Comment by sbesson** [comment]
Superseded by #10 and #11

**Pull request closed without merging**